appendix-examples.md

```diff
@@ -6,6 +6,26 @@ places to deepen understanding.
 
 ## EXAMPLES
 
+### SORMAS
+
+Developed by a consortium of German and Nigerian public health
+research institutions, [SORMAS](https://sormas.org/) is software for
+managing and controlling epidemic outbreaks.  It is used extensively
+in response to Ebola outbreaks in multiple countries.
+
+### Lutece
+
+This [https://www.lutece.paris.fr/](government services portal) was
+developed by the city of Paris, France and has recently become the
+subject of international attention, as they make greater efforts to
+support multi-jurisdictional adoption.
+
+### VITAM
+
+[VITAM](https://www.programmevitam.fr/) is open source project is used
```

Comment:
Consider rephrasing: 
VITAM is an open source project that is used...
or
VITAM is an open source project used...